dataactcore/migrations/env.py

```diff
@@ -1,9 +1,13 @@
 from __future__ import with_statement
 from alembic import context
+# Load all DB tables into metadata object
+from dataactcore.models import domainModels
```

Comment:
Without importing these, the Base metadata object does not get the information about the tables and alembic tries to drop them all.  The point of these imports is to load all the tables into the common Base object they all import, which we then get from the next line "from dataactcore.models import baseModel".